### ISSUE TITLE
The Share button may be hard to read

### ISSUE THREAD

**Issue by DNin01** [LLK/scratch-www#7280]
### The Problem

The Share button has a background with the color `#ffab1a` with text with the color `#ffffff`. The contrast ratio is 1.89, which doesn't meet WCAG color contrast standards.

### Steps to Reproduce

1. Sign into Scratch
2. Open one of your unshared projects
3. Find the Share button on the banner at the top of the page or the menu bar in the editor

### How to Fix it
A dark-ish blue or magenta shade might have higher contrast. We could also make the button light with dark text on it.

### System Details
Source: https://contrastchecker.com/

**Comment by benjiwheeler** [comment]
The site's overall color scheme is being looked at with an eye to accessibility.

**Comment by DNin01** [comment]
How about `hsl(221deg 100% 51%)`?

![The share notification that's over unshared projects, recolored blue](https://user-images.githubusercontent.com/106490990/213312042-d89ca616-121e-4146-a6a8-e7e2e146b9d9.png)
